packages/zeplin-extension-style-kit/values/gradient/gradient.js

```diff
@@ -0,0 +1,94 @@
+import { Gradient as ExtensionGradient } from "@zeplin/extension-model";
+
+import LinearGradient from "./linearGradient";
+import RadialGradient from "./radialGradient";
+import ConicGradient from "./conicGradient";
+
+function generateColorGradient({ r, g, b, a }) {
+    return {
+        type: "linear",
+        from: {
+            x: 0.5,
+            y: 0
+        },
+        to: {
+            x: 0.5,
+            y: 1
+        },
+        colorStops: [{
+            color: { r, g, b, a },
+            position: 0
+        }, {
+            color: { r, g, b, a },
+            position: 1
+        }]
+    };
+}
+
+const GRADIENT_TYPE = Object.freeze({
+    LINEAR: "linear",
+    RADIAL: "radial",
+    ANGULAR: "angular"
+});
+
+class Gradient {
+    constructor(gradientObject, width, height) {
+        const {
+            type,
+            from,
+            to,
+            colorStops
+        } = gradientObject;
+
+        this.type = type;
+
+        switch (type) {
+            case GRADIENT_TYPE.LINEAR:
+                this.gradient = new LinearGradient(colorStops, from, to, width, height);
+                break;
+
+            case "radial":
+                this.gradient = new RadialGradient(colorStops, from, to, width, height);
+                break;
+
+            case "angular":
+                this.gradient = new ConicGradient(colorStops);
+                break;
+
+            default:
+                // No-op
+        }
+    }
+
+    static fromRGBA({ r, g, b, a = 1 }) {
+        return new Gradient(
+            new ExtensionGradient(
```

Comment:
I actually only moved the file, and modified it afterwards, [here's how it was before](https://github.com/zeplin/stylesheet-extensions/pull/10/files#diff-34c16dfc007158fbd3cdf3de58a7f8a6L83). Providing an object won't be enough, since stop colors should be converted to extension models as well.